game_controller.py

```diff
@@ -275,22 +278,4 @@ def run(self):
                 print(f"Game Over! Player {self.game.current_player()} wins! ")
                 break
             print(result)
-        stop_event.set()
-
-
-if __name__ == "__main__":
-    c = GameController(Camera(0), True)
-    while True:
-        # img = c.camera.get_image()
-        img = cv2.imread("DEBUG-skew.png")
-        ships = c.get_ships(img)
-        print([sorted(ship.filled) for ship in ships])
-        break
-        # c.camera.detect_arucos(img.copy())
-        # g = c.get_guess(img, 1)
-        # if g:
-        #     print(g)
-
-        k = cv2.waitKey(5)
-        if k == 27:
-            break
+        self.stop_event.set()
```

Comment:
Right now the game kind of crashes when the game is finished, this should fix it
```suggestion
        self.stop_event.set()
        exit(0)
```